### PR TITLE
Add a web app manifest to the application template

### LINKED ISSUE #166: Feature request: autogenerate Web App Manifest
_As a Polymer CLI user,
when I use `polymer init` to start a new app,
I want the Polymer CLI to generate a [Web App Manifest](https://www.w3.org/TR/appmanifest/),
so that the default scaffolds are one step closer to a true PWA._

Polymer CLI already has a lot of the information necessary to generate a Web App Manifest. It would be really cool if I got one for free when I ran `polymer init`, and if it were already included in the generated `index.html`.

### PR DESCRIPTION
**Pull request by justinfagnani**
Fixes #166

### CODE REVIEW

**Comment by justinfagnani** [comment]
@cdata this is about all the information we have available right now. Things like theme color and icons will likely have to be in a more advanced template, like app-drawer.

File: src/init/application/templates/manifest.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "<%= name %>",
+  "short_name": "<%= name %>",
+  "description": "<%= description %>",
+  "start_url": "/",
+  "display": "standalone",
```
Comment (justinfagnani) [inline src/init/application/templates/manifest.json]: I'm not really sure that "standalone" is the best option...

**Comment by justinfagnani** [comment]
@garlicnation ping

File: src/init/application/templates/manifest.json (hunk 1 above)
Comment (justinfagnani) [inline src/init/application/templates/manifest.json]: Turns out PWAs need "standalone"

File: src/init/application/templates/manifest.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "<%= name %>",
+  "short_name": "<%= name %>",
+  "description": "<%= description %>",
```
Comment (FredKSchott) [inline src/init/application/templates/manifest.json]: need an if statement to make sure description isn't empty string when not provided

Comment (justinfagnani) [inline src/init/application/templates/manifest.json]: done

**Comment by FredKSchott** [comment]
👍 LGTM

**Pull request merged**